bitutils/bitutils_test.go

```diff
@@ -0,0 +1,88 @@
+package bitutils
+
+import (
+	"testing"
+)
+
+// Define some test data
+
+// 0000 0100 1010 0010 0000 0011 1011 0001 0000 0000 0010 1011
+
+var testdata = []byte{0x04, 0xa2, 0x03, 0xb1, 0x00, 0x2b}
+
+func TestParseByte4(t *testing.T) {
+	b, err := ParseByte4(testdata, 21)
```

Comment:
Please use table driven tests and expand tests cases to improve code coverage. A description field would be great to explain the test case. This applies to all parse methods.